common/religion/religions/grond_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_shunned
 	doctrine = doctrine_witchcraft_shunned
 
+	#Magic
+	doctrine = doctrine_light_magic_ignored
+	doctrine = doctrine_shadow_magic_accepted
+	doctrine = doctrine_disorder_magic_accepted
+	doctrine = doctrine_order_magic_approved
+	doctrine = doctrine_life_magic_ignored
+	doctrine = doctrine_death_magic_accepted
+	doctrine = doctrine_elemental_spirit_magic_accepted
```

Comment:
Also, ogres not really well with elements.
https://wow.gamepedia.com/Ogre#The_destruction_of_Goria
Maybe spirit magic should be ignored?

Reply:
Done